Tools/MGCB/BuildContent.cs

```diff
@@ -430,17 +435,25 @@ public void Build(out int successCount, out int errorCount)
                     // Only copy if the source file is newer than the destination.
                     // We may want to provide an option for overriding this, but for
                     // nearly all cases this is the desired behavior.
-                    if (File.Exists(dest))
+                    if (File.Exists(dest) && !Rebuild)
```

Comment:
@cra0zy @nkast @Jjagg 

It made sense to me to make a "Rebuild" command always re-copy the content for copy items.

Agree?